### ISSUE TITLE
attachments and views

### ISSUE THREAD

**Issue by khale** [diderot-edu/diderot-feedback#94]
A couple of complaints I've received from students:

- Apparently they cannot look at their submissions/answers after they've hit submit (even for regular submittable assignments). Presumably they could start a new submission (if I adjust the submission count to be >1) to see their answers, but this would only work up to the due date. 
- You can only attach one picture to an answer. I think in most cases this can be addressed with breaking down problems into more sensible \ask/\sol blocks but it is a complaint I've heard.

**Comment by umutacar** [comment]
Yes they can only see their answers if the solutions or the feedback is published.  We can add an option to allow for viewing solutions.

Re: attachments yes this is correct.  Thank you for feedback.  Will think about allowing multiple.
